src/module.zig

```diff
@@ -25,7 +25,13 @@ const PackedObjectTypeID = packed struct(u16) {
     object_name_id: u6,
 };
 
-pub fn Objects(comptime T: type) type {
+pub const ObjectsOptions = struct {
+    /// If this option is true, the internal field `update` will
+    /// contain a list of bitsets representing each object field
+    track_fields: bool = false,
+};
+
+pub fn Objects(comptime T: type, options: ObjectsOptions) type {
```

Comment:
Since `options` will always be shorter than `T` I think when writing it, let's put options first?

Reply:
I guess we could expose `mach.Objects(T)` and `mach.ObjectsWithOptions(T, options)`

Might be better.. hmm